Resources/Prototypes/Body/Presets/drone.yml

```diff
@@ -0,0 +1,10 @@
+- type: bodyPreset
+  name: "drone"
+  id: DronePreset
+  partIDs:
+    hand 1: LeftHandHuman
+    hand 2: LeftHandHuman
+    hand 3: LeftHandHuman
+    hand 4: RightHandHuman
+    hand 5: RightHandHuman
+    hand 6: RightHandHuman
```

Comment:
human?

Reply:
Is there any reason not to if they aren't gonna be getting dismembered or butchered

Reply:
Gibbing.

Reply:
They explode on death